packages/config/tailwind-preset.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   content: [
     "./pages/**/*.{js,ts,jsx,tsx}",
     "./components/**/*.{js,ts,jsx,tsx}",
-    "../../packages/app-store/**/{components,pages}/**/*.{js,ts,jsx,tsx}",
+    "../../packages/app-store/**/{_components,_pages}/**/*.{js,ts,jsx,tsx}",
```

Comment:
Components and pages folders were renamed and Tailwind process was not scanning them.

Reply:
This is an incomplete change. This is referring to each individual app's pages and components. You're replacing them with shared components. The correct solution should take both into account.